searx/engines/genius.py

```diff
@@ -27,49 +32,54 @@
 
 
 def request(query, params):
-    params['url'] = search_url.format(query=urlencode({'q': query}),
-                                      index='multi',
-                                      page_size=page_size,
-                                      pageno=params['pageno'])
+    params['url'] = search_url.format(
+        query=urlencode({'q': query}),
+        index='multi',
+        page_size=page_size,
+        pageno=params['pageno'],
+    )
     return params
 
 
 def parse_lyric(hit):
     try:
         content = hit['highlights'][0]['value']
-    except:
+    except Exception as e:  # pylint: disable=broad-except
+        logger.error(e, exc_info=True)
         content = ''
     timestamp = hit['result']['lyrics_updated_at']
-    result = {'url': hit['result']['url'],
-              'title': hit['result']['full_title'],
-              'content': content,
-              'thumbnail': hit['result']['song_art_image_thumbnail_url'],
-              'template': 'videos.html'}
```

Comment:
Just note, not a request for change: technically this is right to use `default.html` instead of `videos.html`, but the oscar layout is different (it doesn't change anything for the simple theme).

Reply:
IMO both are missing a result template for "music" hits.